[2024-12-26T06:18:19.857+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-12-26T06:18:19.909+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: api_to_kafka_pipeline.fetch_and_produce_data manual__2024-12-26T06:18:16.682582+00:00 [queued]>
[2024-12-26T06:18:19.930+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: api_to_kafka_pipeline.fetch_and_produce_data manual__2024-12-26T06:18:16.682582+00:00 [queued]>
[2024-12-26T06:18:19.931+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2024-12-26T06:18:19.977+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): fetch_and_produce_data> on 2024-12-26 06:18:16.682582+00:00
[2024-12-26T06:18:20.020+0000] {logging_mixin.py:190} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=366) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-12-26T06:18:20.008+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'api_to_kafka_pipeline', 'fetch_and_produce_data', 'manual__2024-12-26T06:18:16.682582+00:00', '--job-id', '80', '--raw', '--subdir', 'DAGS_FOLDER/api_to_kafka_dag.py', '--cfg-path', '/tmp/tmpsjs2muhq']
[2024-12-26T06:18:20.023+0000] {standard_task_runner.py:72} INFO - Started process 370 to run task
[2024-12-26T06:18:20.023+0000] {standard_task_runner.py:105} INFO - Job 80: Subtask fetch_and_produce_data
[2024-12-26T06:18:20.237+0000] {task_command.py:467} INFO - Running <TaskInstance: api_to_kafka_pipeline.fetch_and_produce_data manual__2024-12-26T06:18:16.682582+00:00 [running]> on host a7d7ef0c0878
[2024-12-26T06:18:20.488+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='api_to_kafka_pipeline' AIRFLOW_CTX_TASK_ID='fetch_and_produce_data' AIRFLOW_CTX_EXECUTION_DATE='2024-12-26T06:18:16.682582+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-12-26T06:18:16.682582+00:00'
[2024-12-26T06:18:20.491+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-12-26T06:18:21.128+0000] {logging_mixin.py:190} INFO - âœ… Data fetched successfully for IBM at 5min interval.
[2024-12-26T06:18:21.130+0000] {logging_mixin.py:190} INFO - ðŸš€ Sending data to Kafka...
[2024-12-26T12:29:25.447+0000] {job.py:229} INFO - Heartbeat recovered after 20484.37 seconds
[2024-12-26T13:54:42.257+0000] {job.py:229} INFO - Heartbeat recovered after 5117.39 seconds
[2024-12-26T16:09:56.084+0000] {job.py:229} INFO - Heartbeat recovered after 665.37 seconds
[2024-12-26T16:10:07.032+0000] {local_task_job_runner.py:346} WARNING - State of this instance has been externally set to up_for_retry. Terminating instance.
[2024-12-26T16:10:07.050+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
[2024-12-26T16:10:07.097+0000] {process_utils.py:132} INFO - Sending 15 to group 370. PIDs of all processes in the group: [370]
[2024-12-26T16:10:07.189+0000] {process_utils.py:87} INFO - Sending the signal 15 to group 370
[2024-12-26T16:11:07.207+0000] {process_utils.py:150} WARNING - process psutil.Process(pid=370, name='airflow task ru', status='sleeping', started='06:18:19') did not respond to SIGTERM. Trying SIGKILL
[2024-12-26T16:11:07.257+0000] {process_utils.py:87} INFO - Sending the signal 9 to group 370
[2024-12-26T16:11:07.286+0000] {process_utils.py:80} INFO - Process psutil.Process(pid=370, name='airflow task ru', status='terminated', exitcode=<Negsignal.SIGKILL: -9>, started='06:18:19') (370) terminated with exit code -9
[2024-12-26T16:11:07.289+0000] {standard_task_runner.py:190} ERROR - ('Job 80 was killed before it finished (likely due to running out of memory)', 'For more information, see https://airflow.apache.org/docs/apache-airflow/stable/troubleshooting.html#LocalTaskJob-killed')
